var/spack/repos/builtin/packages/diffutils/package.py

```diff
@@ -29,7 +29,9 @@ class Diffutils(AutotoolsPackage, GNUMirrorPackage):
     patch('intprops-workaround-nvc-22.1-bug.patch', sha256='146b7021bb0a304a3d1c0638956c4e735c2076d292d238f2806efadc972d99e5', when='@3.8 %nvhpc')
 
     conflicts('%nvhpc', when='@:3.6,3.8:')
-
+    conflicts('%intel@2021:', when='@3.8')
+    conflicts('%intel@19.1.2.254', when='@3.8')
```

Comment:
Can you link to some bug? Or do you have an explanation why these versions don't work?

Reply:
I don't know a good reason, I simply tested compilers with an exhaustive approach with several different versions of each compiler available in spack. If you want the list of those results I can try to make a small word document or whatever works for you, right now I only have them handwritten.

Reply:
@rscohn2 Do you know of any reported bug that might cause this? I am still trying to triage if this is user configuration (like suggested by @rountree) or something broader.